### PR TITLE
[index.md] add docs for new partner destination Skalin

### PR DESCRIPTION
**Pull request by tcgilbert**
### Proposed changes

add docs for new partner destination Skalin

### Merge timing
asap

### Related issues (optional)

n/a

### CODE REVIEW

File: src/connections/destinations/catalog/skalin/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+4. Go to the [Skalin plateform Segment integration page](https://app.skalin.io/#/integration/segment){:target="_blank"}, click on "Connect" button to create configuration, find and copy the **API Key** and "Skalin Client ID" in  Segment connection section.
+5. Enter the **API Key** and **ClientID** in the "Skalin" destination settings in Segment.
+
+## Supported methods
+
+Skalin supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).
+
+### Page
+
+Send [Page](/docs/connections/spec/page) calls to *feature*. For example:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/skalin/index.md:29]: Is it intended to say *feature* here and for the Screen and Track sections?

File: src/connections/destinations/catalog/skalin/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+```obj-c
+[[SEGAnalytics sharedAnalytics] screen:@"Home"];
+```
+
+Segment sends Screen calls to Skalin as a `feature`. 
+
+
+### Identify
+
+Send [Identify](/docs/connections/spec/identify) calls to *identity*. For example:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/skalin/index.md:51]: Is it right to say "calls to *identity*"?

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
@tcgilbert I left some comments that could use some clarification

**Comment by tcgilbert** [comment]
@stayseesong addressed all of your points in the last couple of commits. Please let me know if you have any other questions!

**Review by stayseesong — APPROVED** [review]

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**